src/TLuaInterpreter.cpp

```diff
@@ -10467,7 +10467,7 @@ int TLuaInterpreter::downloadFile(lua_State* L)
     connect(reply, &QNetworkReply::downloadProgress, [=](qint64 bytesDownloaded, qint64 totalBytes) {
         raiseDownloadProgressEvent(L, urlString, bytesDownloaded, totalBytes);
         if (mudlet::debugMode) {
-            TDebug(QColor(Qt::white), QColor(Qt::blue)) << "\n" << bytesDownloaded << "/" << totalBytes << " for file "
+            TDebug(QColor(Qt::white), QColor(Qt::blue)) << "\n" << bytesDownloaded << "/" << totalBytes << " bytes downloaded for file "
```

Comment:
Do we need the `\n` in that line as that is separating this line from the preceding message generated by the `raiseDownloadProgressEvent(...)` or can we not guarantee that that text will be there? If we cannot perhaps we should use:
```suggestion
            TDebug(QColor(Qt::white), QColor(Qt::blue)) << "\n" << Download progress: << bytesDownloaded << "of:" << totalBytes << " bytes for file: "
```